src/main/java/ai/pluggy/client/response/ConnectorType.java

```diff
@@ -11,7 +11,13 @@ public enum ConnectorType {
   @SerializedName("BUSINESS_BANK")
   BUSINESS_BANK("BUSINESS_BANK"),
   @SerializedName("PERSONAL_BANK")
-  PERSONAL_BANK("PERSONAL_BANK");
+  PERSONAL_BANK("PERSONAL_BANK")
```

Comment:
It is missing "," to separate fields in the model. This will break

Reply:
fixed